RxJavaTemplate/app/src/androidTest/java/co/nimblehq/rxjava/repository/TestApiRepositoryImpl.kt

```diff
@@ -0,0 +1,15 @@
+package co.nimblehq.rxjava.repository
+
+import co.nimblehq.rxjava.domain.data.Data
+import co.nimblehq.rxjava.domain.repository.ApiRepository
+import co.nimblehq.rxjava.domain.test.MockUtil
+import io.reactivex.Single
+import java.util.concurrent.TimeUnit
+import javax.inject.Inject
+
+class TestApiRepositoryImpl @Inject constructor() : ApiRepository {
+
+    override fun exampleData(): Single<List<Data>> {
+        return Single.just(MockUtil.dataList).delay(3, TimeUnit.SECONDS)
```

Comment:
delay 3 secs is too long, maybe we can decrease to 300 millisecond 😅

Reply:
300 millisecond is too short 😅 One of the tests (which using `IdlingResource`) failed with this value. So I changed it to 500 for now. 🙏🏼

Reply:
Resolved